src/MultipleInput.php

```diff
@@ -155,7 +161,7 @@ protected function initData()
         }
 
         if ($this->model instanceof Model) {
-            $data = $this->model->hasProperty($this->attribute)
+            $data = $this->model->hasAttribute($this->attribute)
```

Comment:
better to revert to `hasProperty` because simple models don't have attributes, they have only properties